packages/legacy/keys/package.json

```diff
@@ -0,0 +1,113 @@
+{
+    "name": "@solana/keys-legacy",
```

Comment:
Let's keep a consistent naming convention. 🙏 

```suggestion
    "name": "@solana/legacy-keys",
```

Reply:
The idea here is to publish this legacy keys-related stuff at `1.x` and the new stuff at `>2.x`.

Reply:
Oh, the thing that's not obvious here is that the [build script](https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1932/chore-create-custom-publish-strategy-for-legacy-packages?review=PRR_kwDOCLAddc5pYnKy) strips the `-legacy` suffix at build time. The package _actually_ gets published as `@solana/keys` on npm.

Reply:
Ohhhh god, I didn't realize this was what you were doing with these. So this is `@solana/keys@1.x` and the experimental one is `@solana/keys@2.x`?

I guess it kind of makes sense though, it aligns with `@solana/web3.js@1.x`. Does that mean someone who wants to use `SubtleCrypto` and `@solana/keys@2.x` with the legacy Web3.js is smoked?

Reply:
> Does that mean someone who wants to use SubtleCrypto and @solana/keys@2.x with the legacy Web3.js is smoked?

![88hx9g.jpg](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/lE1zD7Jpg6mWNv3djNKZ/d3dce7dd-f97d-4c6e-9ac4-dadd129720ae.jpg)